README.md

```diff
@@ -15,3 +15,13 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+Answer: The wost case time complexity for this mergesort algorithm would be $\Theta$(nlog n).
+1. mergesort iterates through the array where the width grows *2 each iteration = logn
+2. calculating the mid and right indexes = n
+3. swapping elements in the mergeInPlace function = n / 2, n / 2 = n^2/4
```

Comment:
Don't know what you mean here. Where are elements swapped? What does "n/2, n/2" mean?